homeassistant/components/emulated_hue/__init__.py

```diff
@@ -244,6 +247,12 @@ def __init__(self, hass, conf):
             if hidden_value is not None:
                 self._entities_with_hidden_attr_in_config[entity_id] = hidden_value
 
+        # Get whether all non-dimmable lights should be reported as dimmable
+        # for compatibility with older installations.
+        self.lights_all_dimmable = conf.get(
+            CONF_LIGHTS_ALL_DIMMABLE, DEFAULT_LIGHTS_ALL_DIMMABLE
+        )
```

Comment:
```suggestion
        self.lights_all_dimmable = conf[CONF_LIGHTS_ALL_DIMMABLE]
```

Reply:
Unfortunately, this isn't a workable change as some tests aren't using the new standard config. Also likely explains while all this code is still using config.get(...)